uk.co.compucorp.civicrm.tasksassignments/tasksassignments.php

```diff
@@ -409,3 +409,32 @@ function tasksassignments_extensionsPageRedirect()  {
   );
   CRM_Utils_System::redirect($url);
 }
+
+/**
+ * Implements the alterAngular hook so it can modify the template for the
+ * Workflow Create/Edit screen.
+ *
+ * @param \Civi\Angular\Manager $angular
+ * @return \Civi\Angular\ChangeSet
+ */
+function tasksAssignments_civicrm_alterAngular(\Civi\Angular\Manager $angular) {
+  $changeSet = \Civi\Angular\ChangeSet::create('Workflow Modifications');
+
+  $changeSet->alterHtml('~/crmCaseType/edit.html', function (phpQueryObject $doc) {
+    _tasksAssignments_civicrm_alterAngular_change_workflow_help_text($doc);
+  });
+
+  $angular->add($changeSet);
+}
+
+/**
+ * Modifies the help text for the Workflow Create/Edit screen.
+ *
+ * @param phpQueryObject $doc
+ */
+function _tasksAssignments_civicrm_alterAngular_change_workflow_help_text(phpQueryObject $doc) {
```

Comment:
Since this in an helper function, no need to have the hook name in the function name. The function can simply be named `_tasksAssignments_change_workflow_help_text`.

Also please move the hook to the upper part of the file while the helper function can stay where it is. 
If you check the file, you will notice hooks are grouped together at the top while the helper functions come after.